.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   tests:
     name: Tests
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Do we _need_ macos runs? They're 10 times more expensive:
![image](https://user-images.githubusercontent.com/6446452/217221475-61773a1f-dd2f-4506-b559-2228f4fea6f9.png)

Reply:
But the repo is public so we don't have to pay/have limits afaik

Reply:
And yes, we seem to do since otherwise compiling webview fails 😢

Reply:
> GitHub Actions usage is free for standard GitHub-hosted runners in public repositories, and for self-hosted runners.
from [the docs](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration)

Reply:
Thanks for explaining! 👍